purview/models/distgit.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from neomodel import (
+    StructuredNode, UniqueIdProperty, RelationshipTo, StringProperty,
+    DateTimeProperty)
+
+
+class DistGitCommit(StructuredNode):
+    author_date = DateTimeProperty(required=True)
+    commit_date = DateTimeProperty(required=True)
+    id_ = UniqueIdProperty(db_property='id')
+    log_message = StringProperty()
+    sha = StringProperty(required=True)
+    authors = RelationshipTo('.user.User', 'AUTHORED_BY')
+    branches = RelationshipTo('DistGitBranch', 'CONTAINED_BY')
+    committers = RelationshipTo('.user.User', 'OWNED_BY')
+    owners = RelationshipTo('.user.User', 'OWNED_BY')
+    pushes = RelationshipTo('DistGitPush', 'PUSHED_IN')
+    related_bugs = RelationshipTo('.bugzilla.BugzillaBug', 'RELATES_TO')
+    repos = RelationshipTo('DistGitRepo', 'CONTAINED_BY')
+    resolved_bugs = RelationshipTo('.bugzilla.BugzillaBug', 'RESOLVES')
```

Comment:
Should this be `RESOLVED` instead of `RESOLVES`? Same question for the one below.